src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -0,0 +1,719 @@
+============== file: test/test.js ==============
+* Copyright 2016 Google Inc. All rights reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+
+var assert = require('assert');
+var service = require('../src/').v1();
+
+describe('LibraryServiceClient', function() {
+  describe('createShelf', function() {
+    it('createShelfTest', function(done) {
+      var client = service.LibraryServiceClient();
+      // Mock request
+      var shelf = {};
+      var request = {
+          shelf : shelf
+      };
+
+      // Mock response
+      var name = 'name3373707';
+      var theme = 'theme110327241';
+      var internalTheme = 'internalTheme792518087';
+      var expectedResponse = {
+          name : name,
+          theme : theme,
+          internalTheme : internalTheme
+      };
+
+      // Mock Grpc layer
+      client._createShelf = function(actualRequest, options, callback) {
+        assert.equal(actualRequest.shelf, shelf);
```

Comment:
Probably better to use `assert.deepStrictEqual` (https://nodejs.org/api/assert.html#assert_assert_deepstrictequal_actual_expected_message)